TWLight/applications/views.py

```diff
@@ -963,16 +971,17 @@ def get_form(self, form_class=None):
             messages.add_message(
                 self.request,
                 messages.WARNING,
+                # fmt: off
                 # Translators: This message is shown to users when they access an application page of a now Bundle partner (new applications aren't allowed for Bundle partners and the status of old applications cannot be modified)
-                _(
-                    "This application cannot be modified since this "
+                _("This application cannot be modified since this "
                     'partner is now part of our <a href="{bundle}">bundle access</a>. '
                     "If you are eligible, you can access this resource from <a href="
                     '"{library}">your library</a>. <a href="{contact}">'
                     "Contact us</a> if you have any questions.".format(
                         bundle=bundle_url, library=library_url, contact=contact_url
                     )
                 ),
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string